### ISSUE TITLE
READ THIS IF YOU WANT HELP

### ISSUE THREAD

**Issue by Varbin** [dlitz/pycrypto#238]
**tl;dr:** PyCrypto is dead. Switch to PyCryptodome (GitHub: https://github.com/Legrandin/pycryptodome PyPi: https://pypi.python.org/pypi/pycryptodome). **Do not report any new bugs here!**

---

As you may or may not have noticed the last change to PyCrypto was made **three years ago.**

As a result the developers don't react to issues on the issue tracker anymore. If you find problems and want them to be fixed consider following:

As a **user** or **developer** who uses software which requires PyCrypto:
 - Tell this to your application's developer to switch to PyCryptodome and tell them PyCrypto is dead.

As a **developer** who develops/maintains **software depending on PyCrypto**:
 - Consider switching to Pycryptodome - it's a well mantained fork and mostly compatible. If your problems persists report your problem to Pycryptodome's developer.

As a **developer** who wants to develop **new software**:
- *Do not use Pycrypto!* Use Pycrpytodome or cryptography. PyCrypto is old and not maintained.

**Comment by s-weigand** [comment]
Since i was about to write @dlitz (repo owner) an email about this, I was wandering if any maintainer is still around to put this issues text on the repos frontpage, so people will get redirected to `Pycrpytodome` or `cryptography` or even hake a PYPI release with that information. Life happens and projects get deprecated, but i feel sad to see recent PR's to a dead project, it's just a waste of well ment effort. 😢

**Comment by cmcqueen** [comment]
It would be good if GitHub could come up with a general automatic mechanism for this, to automatically figure out when a parent project is dead, and which fork might be a suitable go-to. Ie don't show merely a list of forks, but show an indication of their activity. Ideally, a quick perusal of the list of forks would make it obvious which forks are active.

**Comment by avrajsri** [comment]
pip uninstall crypto
Delete "crypto" Folder [C:\Users\AppData\Local\Programs\Python\Python37\Lib\site-packages]
pip install crypto
pip uninstall pycryptodome
pip install pycryptodome

pip uninstall pycryptodomex
pip install pycryptodomex
pip uninstall cryptography
pip install cryptography
	
	
Rename Folder "crypto" to "Crypto"

Then Use from Crypto.Cipher import AES

**Comment by Iiridayn** [comment]
Also has usability concerns: https://www.ieee-security.org/TC/SP2017/papers/161.pdf published in 2017.
